sdk/core/azure-core/src/main/java/com/azure/core/implementation/serializer/jackson/JacksonAdapter.java

```diff
@@ -117,12 +117,14 @@ public String serialize(Object object, SerializerEncoding encoding) throws IOExc
 
     @Override
     public String serializeRaw(Object object) {
+        final ClientLogger logger = new ClientLogger(JacksonAdapter.class);
         if (object == null) {
             return null;
         }
         try {
             return serialize(object, SerializerEncoding.JSON).replaceAll("^\"*", "").replaceAll("\"*$", "");
         } catch (IOException ex) {
+            logger.warning("Failed to serialize {} to JSON.", object.getClass());
```

Comment:
Log the exception as the last parameter in this.

Reply:
Done.